tools/modules/functions/module_default.sh

```
@@ -0,0 +1,106 @@
+# module_default.sh
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)